tests/test_ophys/test_tools_roiextractors.py

```diff
@@ -609,6 +609,23 @@ def test_none_masks(self):
         assert "pixel_mask" not in plane_segmentation
         assert "voxel_mask" not in plane_segmentation
 
+    def test_invalid_mask_type(self):
+        """Test that an invalid mask_type raises a AssertionError."""
+        segmentation_extractor = generate_dummy_segmentation_extractor(
+            num_rois=self.num_rois,
+            num_frames=self.num_frames,
+            num_rows=self.num_rows,
+            num_columns=self.num_columns,
+        )
+        with assert_raises(AssertionError):
```

Comment:
```suggestion
        with assert_raises(AssertionError) as exception_info:
```

can you add an extra assertion here for the message type, especially since an `AssertionError` is a pretty common type for us